docs/source/hyper-parameter-search.rst

```diff
@@ -420,6 +420,7 @@ This section uses :class:`~dask_ml.model_selection.HyperbandSearchCV`, but it ca
 also be applied to to :class:`~dask_ml.model_selection.IncrementalSearchCV` too.
 
 .. ipython:: python
+   :okwarning:
```

Comment:
We should open issues corresponding to the problems that we are ignoring by this. Essentially to ensure that we have it in the backlog. 


```
informative_idx, beta = dask.compute(
        informative_idx, beta, scheduler="single-threaded"
    )
```